### PR TITLE
remove stale links to discontinued products

### PR DESCRIPTION
**Pull request by mindw**

### CODE REVIEW

**Comment by mattfarina** [comment]
@mindw Thanks for catching these stale related projects. Helm uses a [developers certificate of origin](https://developercertificate.org/) to signify that you can contribute. That's the check that failed. When you git commit it needs a `--signoff` flag. Under the checks tag there are directions on fixing your existing commit. Could you please do that.

File: content/en/docs/community/related.md [hunk 1]
```diff
@@ -140,9 +140,6 @@ Tools layered on top of Helm.
 Platforms, distributions, and services that include Helm support.
 
 - [Kubernetic](https://kubernetic.com/) - Kubernetes Desktop Client
-- [Cabin](https://www.skippbox.com/cabin/) - Mobile App for Managing Kubernetes
-- [Qstack](https://qstack.com)
-- [Fabric8](https://fabric8.io) - Integrated development platform for Kubernetes
```
Comment (mattfarina) [inline content/en/docs/community/related.md]: In the Fabric8 docs they refer to their Helm integration. Why is it being dropped as a related project? Should this one instead be kept? See https://fabric8.io/docs/index.html

Comment (mindw) [reply content/en/docs/community/related.md]: The entire fabric8 project is archived and the docs can go away at a moment's notice. IMHO links to archived projects are worse than no links.

Comment (bacongobbler) [reply content/en/docs/community/related.md]: Interesting: https://github.com/fabric8io/fabric8

**Comment by bacongobbler** [comment]
LGTM pending DCO signoff as @mattfarina mentioned above

**Comment by bacongobbler** [comment]
pinging @mindw

**Comment by bacongobbler** [comment]
fixed in #567

**Pull request closed without merging**